### ISSUE TITLE
Exams break if you delete a channel that had questions in that exam

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#3947]
### Observed behavior

reported by @jtamiace 

Exams break if you delete a channel that had questions in that exam

### Expected behavior

They shouldn't break, but what is the desired UX?

### User-facing consequences

…

### Errors and logs

…

### Steps to reproduce

* Create an exam with exercises from Channel A and Channel B
* Set exam active
* Delete Channel A
* Go to exam in Learn or Coach"

### Context

0.10.0

**Comment by jonboiser** [comment]
I found that is

1. Exam Preview
1. Exam User Report (ExamReportDetailsPage)

that break when content is deleted. The Exam Report Page still seems to work.

**Comment by jonboiser** [comment]
For the Exam User Report, the reason things break is that it requires the actual Exercise/Perseus File + the ContentNode model to render the report fully (with attempts displayed as a filled-in exercise). 

The quick fix for now might be to filter out deleted Exercises from the reports, but it may still be possible to render a text-only version of the log if the Exercise cannot be fully rendered.

**Comment by indirectlylit** [comment]
I've logged a clearinghouse issue to address the larger questions. For now, let's just try and do something reasonable - either filtering completely or (if straightforward) showing a text-only version.

**Comment by indirectlylit** [comment]
this is now addressed with an icon in 0.10.1 - retargeting to 0.11 to add in translated texgt

**Comment by indirectlylit** [comment]
whoops, I forgot to add a user string label and missed string freeze

**Comment by DXCanas** [comment]
Even after #4648, Learn > Class > Exam breaks. Separate issue?

**Comment by indirectlylit** [comment]
@DXCanas would you mind opening an issue for that? All I did with #4648 was change icons to strings
